enterprise/cmd/frontend/internal/batches/resolvers/batch_spec.go

```diff
@@ -614,3 +616,42 @@ func (r *batchSpecResolver) computeCanAdminister(ctx context.Context) (bool, err
 	})
 	return r.canAdminister, r.canAdministerErr
 }
+
+func (r *batchSpecResolver) Mounts(ctx context.Context, args *graphqlbackend.ListBatchSpecMountArgs) (_ graphqlbackend.BatchSpecMountConnectionResolver, err error) {
+	tr, ctx := trace.New(ctx, "Resolver.BatchSpecs", fmt.Sprintf("First: %d, After: %v", args.First, args.After))
```

Comment:
```suggestion
	tr, ctx := trace.New(ctx, "Resolver.BatchSpec.Mounts", fmt.Sprintf("First: %d, After: %v", args.First, args.After))
```

but since this method does pretty much nothing heavy, I think we could just leave tracing out entirely (we don't do that on other fields as well).